python/MetrologyData.py

```diff
@@ -268,11 +269,12 @@ def plot_statistics(self, nsigma=5, title=None, zoffset=0):
                                nsigma), (0.05, 0.8), xycoords='axes fraction')
 
         # Overlay a Gaussian with the same sigma and correct normalization
-        x = np.arange(np.min(dz), np.max(dz), 0.1)
-        binsz = 10*lim/60
```

Comment:
You've deleted `lim` usage here, so it's not needed above if you replace line 261 with
```
xrange = [-5*self.sensor.stdev_filt, 5*self.sensor.stdev_filt]
```
and delete line 260.